contract_variable_qty_timesheet/readme/USAGE.rst

```diff
@@ -0,0 +1,8 @@
+To use this module, you need to:
+
+#. Go to Invoicing > Sales > Contracts and select or create a new contract.
```

Comment:
It should be Go to Invoicing > Customers > Customer Contracts

Reply:
This was solved in the last update.